### ISSUE TITLE
hv.Sankey responsive

### ISSUE THREAD

**Issue by mgsnuno** [holoviz/holoviews#4058]
The responsive options works great (I use it a lot in hv.Bars), would it be possible to allow it in Sankey as well?

Right now if `responsive=True` we get:
```python
WARNING:param.OverlayPlot09653: responsive mode could not be enabled because fixed width and height were specified.
WARNING:param.OverlayPlot09855: responsive mode could not be enabled because fixed width and height were specified.
```
I assume it's because of the underlying construction of the nodes etc which are not exposed to the user.

**Comment by harmbuisman** [comment]
I have the same issue, not matter what I try the Sankey plot always has the same sizing when not setting either width or height. Setting it to responsive=True gives the above error

**Comment by philippjfr** [comment]
An override for now is setting width=None and height=None. I'll fix this for 1.13.0
